src/lowlevel/Video.cpp

```diff
@@ -896,6 +896,35 @@ Rectangle Video::get_viewport() {
 
 /**
  * \brief Converts window coordinates to quest size coordinates.
```

Comment:
Ok no problem, I think it's better indeed.
Should I also do it for the Video::renderer_to_quest_coordinates function, in order to fix #1018 ?